app/javascript/dashboard/components/widgets/BackButton.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <button class="settings back-button" @click.capture="goBack">
+  <button class="back-button" @click.capture="goBack">
```

Comment:
@gitstart Can you give permission to push the changes, or can you make changes here?
There is an SCSS file called `_rtl.scss` [line](https://github.com/chatwoot/chatwoot/blob/76d4c22c2d22f2c88ce59bec6611d016360456a2/app/javascript/dashboard/assets/scss/_rtl.scss#L88) 88 , change the class name to `.header-section.back-button`

```suggestion
  <button class="header-section back-button" @click.capture="goBack">
```

Reply:
Hi @iamsivin, sorry for the delay. 
Thanks for the fix, the PR has now been updated.